src/client/components/LeaderboardList/AmountOfTime.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function AmountOfTime({ timeSpent }) {
+  const hours = Math.floor(timeSpent / 3600);
```

Comment:
Hours. Minutes. Seconds. Good. Declarative programming

Reply:
Try to use a library next time. Like Moment. Regardless of how simple the calculations might be